src/index.html

```diff
@@ -2,20 +2,82 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta 
+    <meta
       name="viewport"
       content="
-        width=device-width, 
-        user-scalable=no, 
-        initial-scale=1.0, 
-        maximum-scale=1.0, 
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
         minimum-scale=1.0
       "
     >
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="attr-req-value"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section class="card" data-qa="card">
+      <div class="image">
+        <img
+        src="images/imac.jpeg"
+        alt="imac.jpeg"
+        class="image__monitor"
+        >
+      </div>
+      <div class="title">
+        <p class="title__text">
+         APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+      </div>
+      <div class="description">
+        <p class="description__text">
+          Product code: 195434
+        </p>
+      </div>
```

Comment:
- you don't need this additional container
- let's use semantic tags
```suggestion
      <h3 class="card__code">
        Product code: 195434
      </h3>
```